File: Hipotenusa/DistanciaP_CS/Distancia.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace DistanciaP_CS
{
    class Distancia : Puntos
    {
        public Distancia(int x1, int x2, int y1, int y2)
        {
            this.x1 = x1;
            this.x2 = x2;
            this.y1 = y1;
            this.y2 = y2;
        }

        public double Calcular()
        {
            double d;
            d = ((Math.Pow(x2 - x1, 2)) + (Math.Pow(y2 - y1, 2)));
            d = (Math.Sqrt(d));
            return d;
        }
    }
}

File: Hipotenusa/Hipotenusa/Distancia.vb
Public Class Distancia
    Inherits Puntos

    Public Sub New(x1 As Integer, x2 As Integer, y1 As Integer, y2 As Integer)
        Me.X1 = x1
        Me.X2 = x2
        Me.Y1 = y1
        Me.Y2 = y2
    End Sub


    Public Function Calcular() As Double
        Dim d As Double
        d = ((Math.Pow(X2 - X1, 2)) + (Math.Pow(Y2 - Y1, 2)))
        d = (Math.Sqrt(d))
        Return d
    End Function
End Class
